src/dbnode/storage/index.go

```diff
@@ -2490,6 +2491,13 @@ func newNamespaceIndexMetrics(
 			"loaded-docs-per-query",
 			tally.MustMakeExponentialValueBuckets(10, 2, 16),
 		),
+		blockPermitWaitTime: scope.Histogram(
+			"block-permit-wait-time",
+			instrument.SparseHistogramTimerHistogramBuckets(),
+		),
+		blockProcessingTime: scope.Histogram("block-processing-time",
+			instrument.SparseHistogramTimerHistogramBuckets(),
```

Comment:
I don't think the default sparse buckets are good for this. they are probably more suited for rpc latency, since the times go up to 8m. we probably want finer grain for a single block (or just remove some buckets to reduce cardinality).

Reply:
unresolve. ok. I had a simpler approach, so I'll use that.